services/autoscaling/tests/unit/test_utils_docker.py

```diff
@@ -790,6 +791,44 @@ async def test_try_get_node_with_name_fake(
     assert received_node is None
 
 
+async def test_find_node_with_name_with_common_prefixed_nodes(
```

Comment:
TIP: what about 
```python

@pytest.mark.acceptance_test("Fixes https://github.com/ITISFoundation/osparc-simcore/issues/4344")
async def test_find_node_with_name_with_common_prefixed_nodes( ... )
```

Reply:
I do not find this to be an "acceptance" test. more of a missing test.